src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionController.kt

```diff
@@ -19,16 +19,15 @@ import org.springframework.security.core.Authentication
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.PostMapping
-import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import java.util.UUID
 
-internal const val IDENTIFICATION_SESSIONS_BASE_PATH = "/api/v1/identification/sessions"
+internal const val IDENTIFICATIONS_BASE_PATH = "/api/v1/identifications"
+internal const val IDENTIFICATIONS_OLD_BASE_PATH = "/api/v1/identification/sessions"
 internal const val TCTOKEN_PATH_SUFFIX = "tc-token"
 
 @RestController
 @Timed
-@RequestMapping(IDENTIFICATION_SESSIONS_BASE_PATH)
 @Tag(
     name = "Identification Sessions",
```

Comment:
You did not touch the tags for the api docs yet. Please coordinate with @UrsKahmann since I talked to him and he planned to do so in his PR.

Reply:
Good point. I'll talk to him